gflownet/envs/README.md

```diff
@@ -0,0 +1,11 @@
+# GFlowNet environments
```

Comment:
Could you add here a paragraph about the meaning of the mask_invalid_actions_forward and mask_invalid_actions_backward, which you told us about today (e.g. that non-invalid doesn't mean the action is valid)? It would be very useful to have this information documented here

Reply:
Yes, addded [here](https://github.com/alexhernandezgarcia/gflownet/pull/138/commits/d974c50a29e1fcbda01859967835cfa9e44fab00).